react-carousel-component/dist/style.css

```diff
@@ -0,0 +1,66 @@
+.hidden {
+  display: none;
+}
+
+.squirtle {
+  background-image: url(../img/squirtle.jpeg);
+  background-size: cover;
+  width: 200px;
+  height: 215px;
+}
+
+.sandshrew {
+  background-image: url(../img/sandshrew.jpeg);
+  background-size: cover;
+  width: 200px;
+  height: 215px;
+}
+
+.umbreon {
+  background-image: url(../img/umbreon.jpeg);
+  background-size: cover;
+  width: 200px;
+  height: 215px;
+}
+
+.mew {
+  background-image: url(../img/mew.jpeg);
+  background-size: cover;
+  width: 200px;
+  height: 215px;
+}
+
+.magikarp {
+  background-image: url(../img/magikarp.png);
+  background-size: cover;
+  width: 200px;
+  height: 215px;
+}
```

Comment:
This is a clever way to do this the problem though is it isn't very flexible. If you wanted to add more items to your carousel you would have to define new classes for every item. A better option would be to store the img url in your array of items, then use that to set the `src` attribute on an `img` tag in your JSX.